test/checkpoint.bats

```diff
@@ -49,6 +57,15 @@ function teardown() {
 	crictl start "$ctr_id"
 	restored=$(crictl inspect --output go-template --template "{{(index .info.restored)}}" "$ctr_id")
 	[[ "$restored" == "true" ]]
+	sleep 1
```

Comment:
if the container is successfully restored, why do we need the sleep?

Reply:
The sleep is needed for the log file test. The container prints something every 0.5 seconds and to make sure there are more lines in the log file then before I am waiting for 1 second.

Reply:
Will do.

Reply:
added